MiniApp/Classes/MiniAppAnalytics.swift

```diff
@@ -0,0 +1,101 @@
+import Foundation
+
+internal enum MiniAppRATEventType: String, CaseIterable {
+    case appear
+    case click
+    case custom
+}
+
+internal enum MiniAppRATEvent: String, CaseIterable {
+    case hostLaunch = "host_launch"
+    case open
+    case close
+
+    func name() -> String {
+        return "mini_app_\(rawValue)"
+    }
+
+    func eType() -> MiniAppRATEventType {
+        switch self {
+        case .hostLaunch:
+            return .appear
+        case .open, .close:
+            return .click
+        }
+    }
+}
+
+internal enum MiniAppAnalyticsParameter: String, CaseIterable {
+    case projectId = "project_id"
+    case miniAppId = "id"
+    case versionId = "version_id"
+    case sdkVersion = "sdk_version"
+
+    func name() -> String {
+        return "mini_app_\(rawValue)"
+    }
+}
+
+// Swift doesn't have load-time initialization so we need
+// this proxy class that is called by LoaderObjC's `load`
+// method.
+public class MiniAppAnalyticsLoader: NSObject {
+    @objc public static func loadMiniAppAnalytics() {
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
+            MiniAppAnalytics.sendAnalytics(event: .hostLaunch)
```

Comment:
I'm not seeing this event come through - I also don't see this get called when I set a breakpoint here.

I noticed that I don't see the `LoaderObjC.m` file showing up in XCode. Could that be related? Do I need to do something else to link this file?

Reply:
fixed, I added the file locally and it did not survived the `pod deintegrate` because pod spec was only getting swift files